src/sfctl/helps/main.py

```diff
@@ -118,3 +118,28 @@
     type: group
     short-summary: Get and delete volume resources
 """
+
+helps['mesh gateway'] = """
+    type: group
+    short-summary: Get and delete mesh gateway resources
+"""
+
+helps['mesh network'] = """
+    type: group
+    short-summary: Get and delete mesh network resources
+"""
+
+helps['mesh code-package'] = """
+    type: group
+    short-summary: Gets the logs for the container of the specified code package of the service replica.
```

Comment:
Gets -> Get

Reply:
of the service replica -> for the given service replica

changing up the wording - suggestion only